optuna/integration/sklearn.py

```diff
@@ -59,16 +50,18 @@
 if not _imports.is_successful():
     BaseEstimator = object  # NOQA
 
+ArrayLikeType = Union[List, np.ndarray, pd.Series, spmatrix]
+OneDimArrayLikeType = Union[List[float], np.ndarray, pd.Series]
+TwoDimArrayLikeType = Union[List[List[float]], np.ndarray, pd.DataFrame, spmatrix]
+IterableType = Union[List, pd.DataFrame, np.ndarray, pd.Series, spmatrix, None]
```

Comment:
After moving ``pandas`` to ``try_import`` block, ``pandas`` might not available here. So please use ``"pd.Seriese"`` and ``"pd.DataFrame"``instead of ``pd.Seriese`` and ``pd.DataFrame``, respectively.